src/Service/CMakeLists.txt

```diff
@@ -5,12 +5,12 @@
 # To make OrbitAsioServer and OrbitGrpcServer available for fuzz-testing
 # the target OrbitServiceLib was introduced. OrbitService still exists,
 # but only builds main.cpp and links to OrbitServiceLib.
-project(OrbitServiceLib)
+project(ServiceLib)
```

Comment:
At some point I'd like to split the library and the app target into separate directories. But that doesn't need to be in this PR. I just wanted to mention it, because I have struggled so far with finding proper names for these two things. If you happen to have a suggestion I'm all ears.

Reply:
ack.